### PR TITLE
Updated CMDC and CDS imports

### PR DESCRIPTION
**Pull request by TomGoBravo**
This PR updates the CMDC field mapping and some CDS warnings.

TODO: the list of fields seems to change fairly regularly and we need a better medium term maintenance plan.

### CODE REVIEW

File: scripts/update_cmdc.py [hunk 1]
```diff
@@ -29,32 +29,54 @@
...
     DEATHS_TOTAL = "deaths_total", CommonFields.DEATHS
+    # Not found 20200616 DEATHS_CONFIRMED = "deaths_confirmed", None
+    # Not found 20200616 DEATHS_SUSPECTED = "deaths_suspected", None
+
+    # STAFFED_BEDS or LICENSED_BEDS?
+    # Not found 20200616 HOSPITAL_BEDS_CAPACITY_COUNT = "hospital_beds_capacity_count", CommonFields.STAFFED_BEDS
     HOSPITAL_BEDS_IN_USE_COVID_CONFIRMED = "hospital_beds_in_use_covid_confirmed", None
+    # Not found 20200616 HOSPITAL_BEDS_IN_USE_COVID_NEW = "hospital_beds_in_use_covid_new", None
     HOSPITAL_BEDS_IN_USE_COVID_SUSPECTED = "hospital_beds_in_use_covid_suspected", None
+    # Not found 20200616 HOSPITAL_BEDS_IN_USE_ANY = "hospital_beds_in_use_any", None
```
Comment (igorkofman) [inline scripts/update_cmdc.py:54]: i'm most interested in this value

File: scripts/update_cmdc.py [hunk 2]
```diff
@@ -29,32 +29,54 @@
...
+    # Not found 20200616 ACTIVE_TOTAL = "active_total", None
+    CASES_TOTAL = "cases_total", CommonFields.CASES
+    # Not found 20200616 CASES_CONFIRMED = "cases_confirmed", None
+    # Not found 20200616 RECOVERED_TOTAL = "recovered_total", CommonFields.RECOVERED
     DEATHS_TOTAL = "deaths_total", CommonFields.DEATHS
+    # Not found 20200616 DEATHS_CONFIRMED = "deaths_confirmed", None
+    # Not found 20200616 DEATHS_SUSPECTED = "deaths_suspected", None
+
+    # STAFFED_BEDS or LICENSED_BEDS?
+    # Not found 20200616 HOSPITAL_BEDS_CAPACITY_COUNT = "hospital_beds_capacity_count", CommonFields.STAFFED_BEDS
```
Comment (igorkofman) [inline scripts/update_cmdc.py:50]: secondarily interested in this one

File: scripts/update_cmdc.py (hunk 1 above)
Comment (TomGoBravo) [reply scripts/update_cmdc.py:54]: There are a 4 so far:
```
hospital_beds_in_use_any = df.set_index('fips dt'.split()).loc[:,"hospital_beds_in_use_any"].dropna()

fips  dt        
31    2020-06-11    2267.0
      2020-06-13    2247.0
      2020-06-14    2206.0
      2020-06-15    2214.0
Name: hospital_beds_in_use_any, dtype: float64

```

File: scripts/update_cmdc.py (hunk 2 above)
Comment (TomGoBravo) [reply scripts/update_cmdc.py:50]: There are more of this:
```
df.set_index('fips dt'.split()).loc[:,"hospital_beds_capacity_count"].dropna()
fips  dt        
6073  2020-03-29       0.0
      2020-03-30       0.0
      2020-03-31     268.0
      2020-04-01     268.0
      2020-04-02     268.0
                     ...  
      2020-06-12     268.0
31    2020-06-13    3883.0
6073  2020-06-13     268.0
31    2020-06-14    3903.0
      2020-06-15    3874.0
Name: hospital_beds_capacity_count, Length: 81, dtype: float64
```

Does it go to CommonFields STAFFED_BEDS or LICENSED_BEDS?

**Pull request merged**